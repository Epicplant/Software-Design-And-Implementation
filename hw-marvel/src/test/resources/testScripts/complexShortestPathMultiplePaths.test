# Stress tests pathfinding to make sure it works for larger data sets with many cycles and different paths.

# Loads a new graph from a csv file
LoadGraph heroes complexHeroGraph.csv

# Finds shortest path between heroes
FindPath heroes A I

# Finds shortest path between heroes
FindPath heroes C H

# Finds shortest path between heroes
FindPath heroes I D