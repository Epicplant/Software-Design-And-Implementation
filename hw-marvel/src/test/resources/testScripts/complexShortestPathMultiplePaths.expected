# Stress tests pathfinding to make sure it works for larger data sets with many cycles and different paths.

# Loads a new graph from a csv file
loaded graph heroes

# Finds shortest path between heroes
path from A to I:
A to H via 1
H to F via 5
F to I via 6

# Finds shortest path between heroes
path from C to H:
C to B via 2
B to H via 1

# Finds shortest path between heroes
path from I to D:
I to E via 6
E to C via 3
C to D via 2